### DISCUSSION TITLE
PCL2是用什么软件开发的?

### DISCUSSION THREAD

**Discussion by kaifazheWangHaoRan** [Meloong-Git/PCL#1833] [讨论]
我想问一下，请问PCL2是用什么软件开发的?

**Comment by Big-Cake-jpg** [comment]
Microsoft Visual Studio

版本我不知道，不过主流应该是 2019 或者 2022。

**Comment by MoYuan-CN** [comment]
你指的是什么？
底层框架：WPF XML
代码语言：Visual Basic .Net
IDE嘛。。。Visual Studio 吧？
